SOURCES/anicorn

```diff
@@ -156,6 +156,14 @@ checkPIDFile() {
     log "[ERROR] Can't extract path to PID file from Unicorn config"
     exit 1
   fi
+
+  local pid
+
+  pid=$(getPID)
```

Comment:
I think we also should check if that PID is PID of Unicorn and return empty PID otherwise. Because if somebody has been killed Unicorn process and there is a lot of spawning processes, a system can run a process with this PID.

Could you please add this check?